chajda/tsquery/augments.py

```diff
@@ -53,114 +58,215 @@ def augments_gensim(lang, word, config=Config(), n=5):
 
     return words
 
-import os
-import fasttext
-import fasttext.util
-from annoy import AnnoyIndex
+
+################################################################################
+# fasttext
+################################################################################
 
 # path used for storing/loading fasttext models and annoy indices
 fasttext_annoy_data_path = os.path.dirname(__file__)
 
+# dictionary used for storing languages as keys, and corresponding loaded fasttext models as values
 fasttext_models = {}
+
+
+def load_fasttext(lang):
+    '''
+    Downloads and loads a fasttext model for a particular language,
+    storing the fasttext model object in the fasttext_models dictionary.
+    Due to the size of fasttext models (>1gb), and the loading time,
+    we do not load all models on program startup.
+    Instead, this function is used to load them lazily as needed.
+    lang: language passed from augments_fasttext
+
+    NOTE:
+    The doctest below has been commented out due to requiring more memory than github actions allows for.
+    # >>> load_fasttext_model('ja') or fasttext_models['ja'] != None
```

Comment:
can we change these from japanese to one of the lower resource languages so that the doctests will actually run?

Reply:
yeah thats a good idea